code/modules/holiday/holidays.dm

```diff
@@ -67,10 +70,11 @@
 
 /datum/holiday/valentines
 	name = VALENTINES
-	begin_day = 9 //6 days early
+	begin_day = 14
```

Comment:
fair enough, I'll dial it back a bit. We still don't need a whole week of valentines day though.